libs/datamodel/core/tests/validation/attributes/schema/repeated_schema_attribute.prisma

```diff
@@ -0,0 +1,37 @@
+datasource testds {
+    provider = "postgresql"
+    url = env("TEST_DATABASE_URL")
+    schemas = ["public", "private"]
+}
+
+generator js {
+    provider = "prisma-client-js"
+    previewFeatures = ["multiSchema"]
+}
+
+model Test {
+  id Int @id
+  @@schema("public")
+}
+
+enum toggle {
+  on
+  off
+
+  @@schema("public")
+  @@schema("private")
+}
+
+
+// [1;91merror[0m: [1mAttribute "@schema" is defined twice.[0m
+//   [1;94m-->[0m  [4mschema.prisma:21[0m
+// [1;94m   | [0m
+// [1;94m20 | [0m
+// [1;94m21 | [0m  [1;91m@@schema("public")[0m
+// [1;94m   | [0m
+// [1;91merror[0m: [1mAttribute "@schema" is defined twice.[0m
+//   [1;94m-->[0m  [4mschema.prisma:22[0m
+// [1;94m   | [0m
+// [1;94m21 | [0m  @@schema("public")
+// [1;94m22 | [0m  [1;91m@@schema("private")[0m
+// [1;94m   | [0m
```

Comment:
Shouldn't this be "can only defined" once? Otherwise this is not covered when using 3, 4 or more times or at least confusing. (But I see this is not a new error message, so possibly unrelated)

Reply:
Good point — it's a generic error, we would have the same if you put two `@@map`s on a model, but I think we can change it everywhere, you're right.